src/app/components/DesktopHeaderNav/DesktopHeaderNav.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from "react";
+import Link from "next/link";
+import { DesktopHeaderNavProps } from "./DesktopHeaderNav.types";
+import { NavIconButton } from "../Header/Header.types";
+import IconButton from "../IconButton/IconButton";
+import MainNav from "./MainNav";
+
+function DesktopHeaderNav({
+  navLabels,
+  isMobileMenuOpen,
+  toggleMobileMenu,
+  isActiveLink,
+  getActiveLink,
+  navIconButtons,
+}: DesktopHeaderNavProps): ReactElement {
```

Comment:
And with the type ReactElement I suppose it was also linter
stupid airbnb config